GPX/GPXRoot.h

```diff
@@ -37,7 +38,10 @@
 @property (strong, nonatomic) NSString *creator;
```

Comment:
I wanted to change as little as possible as part of this PR. We can probably mark them all readonly, as we don't use them. Given that this is a parser, there should be no use in setting different values externally.